main.go

```diff
@@ -21,7 +21,9 @@ func main() {
 		log.Fatal("You must provide an auth token in `with.userToken` must be supplied")
 	}
 
-	a, err := action.NewAction(apiToken, os.Getenv("INPUT_ORGANIZATIONID"), projectName)
+	enableBGP := os.Getenv("INPUT_ENABLEBGP") == "true"
+
+	a, err := action.NewAction(apiToken, os.Getenv("INPUT_ORGANIZATIONID"), projectName, enableBGP)
```

Comment:
I guess the action fails when project is created and bgp-enabling fails. Is there any recovery needed after this? Maybe project removal? I'm not sure what's the protocol in the github actoins.

Reply:
I started looking into addressing this, and I think it would require a fair amount of refactoring.  The behavior of the BGP enablement request added in this PR seems to match that of the API calls to create SSH keys and API keys for the project:  in those cases as well, if the call fails, the action exits before outputting any project properties, and a sweeper action would not be able to clean up after the project.  I've created a separate issue to refactor the action so that, if it successfully created a project but fails later, it outputs the project ID so that a sweeper action can clean it up.